src/main/java/module/training/Skills.java

```diff
@@ -16,30 +16,33 @@
     public static int getSkillAtPosition(int position) {
         switch (position) {
             case 0:
-                return PlayerSkill.KEEPER;
```

Comment:
Have you used same definition as per the CHPP API ? I mean is 0 really for form and not for keeper ?

Reply:
@akasolace that method is for order in list detail panel "0" is refer to first position in following screen (after was keeper, after this PR is form):
![image](https://user-images.githubusercontent.com/15519907/70064470-e1ea1880-15e9-11ea-9f9d-e4cc920662d1.png)

Reply:
ok perfect thank you